scripts/modify-layers-pdok-ogcapi.py

```diff
@@ -78,21 +85,30 @@ def retrieve_layers_from_oat_endpoint(urls = []):
             "service_md_id": service_md_id,
         }
         oat_layers.append(tile_object)
-    return oat_layers 
+    return oat_layers
 
-def retrieve_layers_from_oaf_endpoint(urls = []):
+
+def retrieve_layers_from_oaf_endpoint(urls=[]):
     oaf_layers = []
     for url in urls:
         url_layer = []
         url_info = requests.get(url).json()
-        service_title = url_info['title'] if 'title' in url_info else url.split('/')[-1]
-        service_abstract = url_info['description'] if 'description' in url_info else  "Geen abstract gevonden"
-        service_type = 'api features'#"oapif"
+        service_title = url_info["title"] if "title" in url_info else url.split("/")[-1]
+        service_abstract = (
+            url_info["description"]
+            if "description" in url_info
+            else "Geen abstract gevonden"
+        )
+        service_type = "api features"  # "oapif"
         collection_json = requests.get(url + "/collections").json()
         for collection in collection_json["collections"]:
             collection_name = collection["id"]
             collection_title = collection["title"]
-            collection_abstract = collection["description"] if "description" in collection else "Geen abstract gevonden"
+            collection_abstract = (
+                collection["description"]
+                if "description" in collection
+                else "Geen abstract gevonden"
```

Comment:
Ik zie dit nu, misschien beter om `"Geen abstract gevonden"` te veranderen in `""`